#ifndef __SERIESSETTINGS_HPP__
#define __SERIESSETTINGS_HPP__

#include <QtGui>
#include <QDialog>
#include <QFileDialog>

#include "DirectoryList.hpp"


//class SeriesSettings : public QDialog
//{
//Q_OBJECT

//public:
//  SeriesSettings(DirectoryList* directoryList);

//  // void           SetName(const QString& name);
//  // const QString& GetName() const { return series_; }

//  // SeriesID       GetSeriesID() const { return seriesID_; }

//private slots:
//  void AddDirectory();
//  void RemoveDirectory();

//private:
//  void ConnectSignalSlots();

//private:
//  Ui_SeriesSettings ui_;
//  // QString  series_;
//  // SeriesID seriesID_;

//  DirectoryList* directoryList_;
//};







#endif // __SERIESSETTINGS_HPP__
